docs/src/index.md

```diff
@@ -124,16 +126,19 @@ that can be opened with any image viewer:
 
 !["solution_000040_scalar_resized"](assets/solution_000040_scalar_resized.png)
 
-The method `Trixi.run(...)` expects a single string argument with the path to a
-Trixi parameter file. To quickly see Trixi in action, `default_example()`
-returns the path to an example parameter file with a short, two-dimensional
-problem setup. A list of all example parameter files packaged with Trixi can be
+The method `trixi_include(...)` expects a single string argument with the path to a
+Trixi elixir, i.e.. a text file containing Julia code necessary to set up and run a
+simulation. To quickly see Trixi in action, `default_example()`
+returns the path to an example elixir with a short, two-dimensional
+problem setup. A list of all example elixirs packaged with Trixi can be
 obtained by running `get_examples()`. Alternatively, you can also browse the
-[`examples/`](https://github.com/trixi-framework/Trixi.jl/examples/) subdirectory. If you want to
-modify one of the parameter files to set up your own simulation, download it to
-your machine, edit the configuration, and pass the file path to `Trixi.run(...)`.
+[`examples/`](https://github.com/trixi-framework/Trixi.jl/examples/) subdirectory.
+If you want to modify one of the elixirs to set up your own simulation,
+download it to your machine, edit the configuration, and pass the file path to
+`trixi_include(...)`.
 
 ### Example: Running a simulation with Trixi
+<!-- TODO: Taal docs, create new version of this asciicast -->
```

Comment:
This is the TODO note for you, @sloede :wink:

Reply:
Haha, ok. I'll make a note of it.